rest-server/src/components/project/projectSQLHelpers.ts

```diff
@@ -1,13 +1,15 @@
-export const getProjects = (userId: number) => {
+export const getProjectsHelper = ({ userId }: { userId: number }): string => {
   return `
-    SELECt * FROM projects
+    SELECt id, title, description, collaborators, user_id 
+    FROM projects
     WHERE user_id=${userId}
-  `
+  `;
 };
 
-export const postProject = (title: string, description: string, collaborators: string, userId: number) => {
+export const postProjectHelper = ({ title, description, collaborators, userId }: { title: string, description: string, collaborators: string, userId: number }): string => {
```

Comment:
should make an interface for projects, so it will look cleaner and we can use it in other places. for example

export project interface = { title: string, description: string, collaborators: string, userId: number }
export const postProjectHelper = ({ title, description, collaborators, userId }: Project)